app/sites/all/themes/dtheme/js/script.js

```diff
@@ -12,30 +12,40 @@
 // - http://www.adequatelygood.com/2010/3/JavaScript-Module-Pattern-In-Depth
 (function ($, Drupal, window, document, undefined) {
 
-
 // To understand behaviors, see https://drupal.org/node/756722#behaviors
-Drupal.behaviors.my_custom_behavior = {
-  attach: function(context, settings) {
-
-    // Place your code here.
-
-
-
-
+  Drupal.behaviors.my_custom_behavior = {
+    attach: function (context, settings) {
+      // Place your code here.
       $('#mobile-menu-links li.expanded').prepend('<div class="menu-item-expand-link"><span>+</span></div>');
 
-      $("#mobile-menu-links li.expanded .menu-item-expand-link").live( "click", function() {
-          $(this).parent('li').toggleClass('open');
+      $("#mobile-menu-links li.expanded .menu-item-expand-link").live("click", function () {
+        $(this).parent('li').toggleClass('open');
       });
-
-      $(".mobile-menu-button").live( "click", function() {
-          $('#mobile-menu-links').toggleClass('open');
+      $(".mobile-menu-button").live("click", function () {
+        $('#mobile-menu-links').toggleClass('open');
+      });
+    }
+  };
+
+  Drupal.behaviors.fb_resizer = {
+    attach: function (context, settings) {
+      fb_iframe_resize();
+      $(window).on('resize', function () {
+        fb_iframe_resize();
```

Comment:
Jak chcesz zrobić to ładniej to wystarczy dać  `$(window).on('resize', fb_iframe_resize); `